### DISCUSSION TITLE
Guide : Effect of improved mask

### DISCUSSION THREAD

**Discussion by glucauze** [glucauze/sd-webui-faceswaplab#50] [Guide/Doc]
Here are the effect of improved mask (gray color) : 

Without : 

![notimp](https://github.com/glucauze/sd-webui-faceswaplab/assets/137925069/f3648063-d61a-4db8-bfef-798bc65f4657)

With :

![imp](https://github.com/glucauze/sd-webui-faceswaplab/assets/137925069/03b3b35f-3815-42c4-a9c6-2cd7e3f74d71)

Result : 

![swapped](https://github.com/glucauze/sd-webui-faceswaplab/assets/137925069/5b13e5ed-d2f0-48ff-9e95-a9ca94a3ac5d)

Without : 
![no_improved_mask](https://github.com/glucauze/sd-webui-faceswaplab/assets/137925069/fd26e16b-4b26-4379-a127-81c4967a8400)

With : 

![improved_mask](https://github.com/glucauze/sd-webui-faceswaplab/assets/137925069/fc5a2e3d-a865-4759-aa00-42ea035ad98f)

See also discussion here : https://github.com/glucauze/sd-webui-faceswaplab/discussions/29

**Comment by cocopete** [comment]
Interesting thing with face swaps I've noticed that the creator may notice more details than an outsider/casual observer.  From my perspective, the results look the same.  I've made faceswaps before that I thought looked great, but when another person looked at them they didn't think it looked like whoever it was supposed to be.

Has anyone noticed this phenomenon before?
